tests/foreman/cli/test_discoveredhost.py

```diff
@@ -836,3 +836,63 @@ def test_positive_update_discover_hostname_settings(self):
 
         :CaseImportance: High
         """
+
+    @skip_if_bug_open('bugzilla', 1572947)
+    @tier3
+    @upgrade
+    def test_positive_provision_pxe_host_with_parameters(self):
+        """Provision the pxe-based BIOS discovered host with host parameters from cli using
+        SYSLINUX loader
+
+        :id: 4b315fe1-2eba-4e59-bd0a-9d16ce319ce2
+
+        :Setup:
+            1. Create a BIOS VM and set it to boot from a network
+
+        :steps:
+            1. Build a default PXE template
+            2. Configure HostGroup to be used for provisioning the discovered host
+               after discovery
+            3. PXE Boot the VM (from Network)
+            4. Wait for Host to be discovered by Satellite
+            5. Provision the discovered host with host parameters
+
+        :expectedresults:
+            1. Ensure host appeared in Discovered Hosts on satellite
+            2. Ensure the host is provisioned with correct and all the host parameters
+               given during provisioning
+            3. Ensure the entry from discovered host list disappeared
```

Comment:
As you mentioned above-Discovered Host List, but you check on info command

Reply:
Cool .. will update the docstring.